CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 ## 6.35.0 (Unreleased)
 
+BREAKING CHANGES:
+* metastore: remove non-functioning `tags` field from `google_dataproc_metastore_service`. It was introduced in v6.31.0 but the feature was not yet GA. ([#22636](https://github.com/hashicorp/terraform-provider-google/pull/22636))
+* redis: remove non-functioning `tags` field from `google_redis_instance `. It was introduced in v6.31.0 but the feature was not yet GA. ([#22637](https://github.com/hashicorp/terraform-provider-google/pull/22637))
+
```

Comment:
```suggestion
```
These are beta-only features. Are these entries needed in the GA provider? However, given they're breaking changes, we could probably be extra cautious and keep them in both.

Reply:
We're removing these features from the GA provider because they're beta-only. So the breaking change is only in the GA provider.

Reply:
Okay, it seems you're suggesting the same in https://github.com/hashicorp/terraform-provider-google-beta/pull/9956#discussion_r2087223142. Removed in beta.